lib/svg_sprite.rb

```diff
@@ -389,7 +387,7 @@ def self.all_icons(theme_id = nil)
         .merge(theme_icons(theme_id))
         .merge(custom_icons(theme_id))
         .delete_if { |i| i.blank? || i.include?("/") }
-        .map! { |i| process(i.dup) }
+        .map! { |i| i.strip }
```

Comment:
If you want can use the shorthand version. 🙂 

```suggestion
        .map!(&:strip)
```